plugins/notifications/src/components/NotificationsSideBarItem/NotificationsSideBarItem.tsx

```diff
@@ -29,9 +29,21 @@ import { notificationsApiRef } from '../../api';
 export const NotificationsSidebarItem = (props?: {
   webNotificationsEnabled?: boolean;
   titleCounterEnabled?: boolean;
+  className?: string;
+  icon?: IconComponent;
+  text?: string;
+  disableHighlight?: boolean;
+  noTrack?: boolean;
 }) => {
-  const { webNotificationsEnabled = false, titleCounterEnabled = true } =
-    props ?? { webNotificationsEnabled: false, titleCounterEnabled: true };
+  const {
+    webNotificationsEnabled = false,
+    titleCounterEnabled = true,
+    className,
+    icon = NotificationsIcon,
+    text = 'Notifications',
+    disableHighlight,
+    noTrack,
+  } = props ?? { webNotificationsEnabled: false, titleCounterEnabled: true };
```

Comment:
What about:

```suggestion
{
   webNotificationsEnabled = false,
    titleCounterEnabled = true,
        icon = NotificationsIcon,
    text = 'Notifications',
    ...restProps
  } = props ?? { webNotificationsEnabled: false, titleCounterEnabled: true };
```

And later:
```
<SidebarItem
  .....
  text={text}
  {...restProps}
```